mvpa2/base/externals.py

```diff
@@ -438,7 +445,7 @@ def __check(name, a='__version__'):
         exec "v = %s.%s" % (name, a)
         # it might be lxml.etree, so take only first module
         versions[name.split('.')[0]] = SmartVersion(v)
-    except Exception, e:
+    except Exception as e:
```

Comment:
oh -- pushed one more commit in docs to remove reference to 2.5 (in installation.rst) but may be there is more?  we have dumped 2.5 in 2012 iirc ;)